### DISCUSSION TITLE
Make Text Smaller in a Section / List

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#3696] [Q&A]
I have the following:

```md
### NumPy

 - Generating a random number.
 - Generating array of samples.
 - Visualizing data.
```

I want to control the relative font size of the whole text. For instance, make it a bit smaller.

I tried:

```md
::: {.smaller}

### NumPy

 - Generating a random number.
 - Generating array of samples.
 - Visualizing data.

:::
```

Yet it seems to cause issues (It behaves like a new slide an then there is a bug which jumps to the end).

Is there a way to control the text size of a section like that?

**Comment by mcanouil** [comment]
```md
---
title: "Untitled"
format: revealjs
---

## NumPy {.smaller}

 - Generating a random number.
 - Generating array of samples.
 - Visualizing data.


## NumPy

 - Generating a random number.
 - Generating array of samples.
 - Visualizing data.
```

You can search for "smaller" in <https://quarto.org/>.

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/8896044/208298361-4cb08997-e7c4-405b-a8c9-ee9dccb11c38.png">


PS: please take the habit to provide a complete minimal reproducible example.

**Reply by RoyiAvital** [reply]
But this is 2 different slides. I want sections within the same slide (Hence the use of 3 `#`).

**Reply by mcanouil** [reply]
You did not ask for that and your example did not display that.
If you expect others to guess your intent, good luck!

You have to be explicit and provide full example if you want an actual answer next time.

**Reply by mcanouil** [reply]
````md
---
title: "Untitled"
format: revealjs
---

## Slide

### H1

::: {style="font-size: 50%;"}

- Generating a random number.
- Generating array of samples.
- Visualizing data.

:::

### H2

- Generating a random number.
- Generating array of samples.
- Visualizing data.

````

<img width="664" alt="image" src="https://user-images.githubusercontent.com/8896044/208318088-81d1f6a8-ca43-4d79-8758-a8654e359036.png">


---

@cscheid I think there is a parsing issue with H3 headers inside fenced divs here. See example below.

````md
---
title: "Untitled"
format: revealjs
---

## Slide

::: {style="font-size: 50%;"}

### H1

- Generating a random number.
- Generating array of samples.
- Visualizing data.

:::

### H2

- Generating a random number.
- Generating array of samples.
- Visualizing data.
````

<img width="728" alt="image" src="https://user-images.githubusercontent.com/8896044/208318147-5bc143fb-9d3f-4c16-9622-766582dd7f28.png">

**Reply by RoyiAvital** [reply]
@mcanouil, Indeed there is a bug. See my original post. It also jumps straight to the end of the presentation.

**Reply by RoyiAvital** [reply]
@mcanouil , I thought I was explicit with writing `section`. I guess it is ambiguous and some see it as a slide?

Sorry, I will try to make it more explicit next time.

**Reply by mcanouil** [reply]
The best way to ask for help is to provide a complete example because:

1. It shows exactly what you tried or are trying to achieve.
2. It allows to discard errors/typos elsewhere in the document.
3. It saves time to the person trying to help, i.e., he/she does not have to rewrite the document to make it work by guessing what was inside.

For short, it helps others to help you. It's a win-win situation.

**Comment by sorenwacker** [comment]
For some reason this does not work with code blocks.

**Reply by mcanouil** [reply]
>  does not work 

What does it mean exactly? Which Quarto version are you using?
The thread is more than a year old and Bootstrap and Reveal.js have been upgraded quite few times since then.

Could you share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.


---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
